protocols/DNSADDR.md

```diff
@@ -36,3 +36,52 @@ So, `/dnsaddr/bootstrap.libp2p.io` resolves to (at least) four multiaddrs:
 /ip4/147.75.83.83/tcp/4001/ipfs/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb
 /ip6/2604:1380:2000:7a00::1/tcp/4001/ipfs/QmbLHAnMoJPWSCR5Zhtx6BHJX9KiKNN6tpvbUcqanj75Nb
 ```
+
+## Suffix matching
```

Comment:
Hard space is [breaking markdown renderer](https://user-images.githubusercontent.com/157609/68666614-8675bf00-0544-11ea-822f-648cea985b08.png),  replaced it with regular one :upside_down_face: 
```suggestion
## Suffix matching
```